packages/templates/src/templates/Cargo.toml

```diff
@@ -0,0 +1,4 @@
+[workspace]
+members = [
```

Comment:
what's `lib` directory?

Reply:
I suggest not `lib/`  but `crates/`.
Because contracts are also lib package as you can see here https://github.com/swanky-dapps/dex/blob/cf49b3e2f9d0b43f07071ced71e9461363162986/uniswap-v2/contracts/pair/Cargo.toml#L16.